src/control/cmd/daos_agent/mgmt_rpc.go

```diff
@@ -165,3 +172,8 @@ func (mod *mgmtModule) handleGetAttachInfo(reqb []byte, pid int32) ([]byte, erro
 
 	return mod.aiCache.getResponse(numaNode)
 }
+
+func (mod *mgmtModule) handleDisconnect(reqb []byte, pid int32) ([]byte, error) {
```

Comment:
What is the plan for adding a implementation for this handler?  Is that the subject of a future PR?

Reply:
Yes it is implemented in the next PR. To be honest I'm not sure why i added this here and not in the next one but its how it ended up unfolding.